tools/val/val.cpp

```diff
@@ -68,6 +68,8 @@ NOTE: The validator is a work in progress.
                                    be allowed by the target environment.
   --allow-offset-texture-operand   Allow use of the Offset texture operands where it would otherwise not
                                    be allowed by the target environment.
+  --allow-non-32-bit-bases         Allow use of non-32 bit for the Base operand where it would otherwise
+                                   not be allowed by the target environment.
```

Comment:
This is not a great name or description. `--allow-vulkan-32-bit-bitwise` would be better with a description that it allows certain bitwise instructions to use non-32-bit base operands in Vulkan environments.

Reply:
agree, this is a better name, updated